Properly setup __name__ on unary refs functions.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #79178

My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

ezyang committed:
Properly setup __name__ on refs functions.

My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Properly setup __name__ on refs functions."


My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Properly setup __name__ on unary refs functions."


My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

[ghstack-poisoned]

mruberry commented on torch/_refs/__init__.py:
```diff
@@ -591,74 +575,68 @@ def positive(a: TensorLikeType) -> TensorLikeType:
 
+@_make_elementwise_unary_reference(ELEMENTWISE_TYPE_PROMOTION_KIND.ALWAYS_BOOL)
+def signbit(a):
+    return prims.sign(a)
```
signbit

mruberry reviewed (approved):
Stamp to unblock but test failures are real (see inline comment, `refs.signbit` is calling `sign`)

ezyang committed:
Update on "Properly setup __name__ on unary refs functions."


My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Properly setup __name__ on unary refs functions."


My hands hurt now.  Yes I could have added type annotations if you care do it
yourself.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

[ghstack-poisoned]

ezyang commented:
@pytorchbot merge -g
